modules/exploits/unix/webapp/pfadmin_protected_alias.rb

```diff
@@ -0,0 +1,141 @@
+
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
```

Comment:
exploits give shells, believe this should actually be an aux module. @wvu-r7 is that correct?